api_documentation.md

```diff
@@ -0,0 +1,57 @@
+# API Documentation
+
+> Note: all endpoints return JSON
+
+### /api/login
+*method:* **POST**  
+*parameters:* **username**, **sim_id**, **password**  
+*description:* creates a user entry in the database and compares the entered password with the simulation password
+*onsuccess returns:* the user model  
+
+### /api/simulations
+*method:* **GET**  
+*parameters:* **none**  
+*description:* fetches all the simulations
+*onsuccess returns:* list of simulations in the database  
+
+### /api/pages  
+*method:* **POST**  
```

Comment:
I'm not sure that's in-line with the HTTP request verb specification, but it should be fine, especially since we don't have to worry about users refreshing their POSTed browser pages :-)